src/modules/claims/claims.service.ts

```diff
@@ -392,8 +398,16 @@ export class ClaimsService {
     const strippedClaimData = this.stripClaimData(claimData);
     const { claimType: role, claimTypeVersion: version } = claimData;
 
+    const defaultValidityPeriod = roleDefinition?.['defaultValidityPeriod'];
+    const roleDefinitionExpirationTimestamp = defaultValidityPeriod
+      ? Date.now() + defaultValidityPeriod
+      : undefined;
```

Comment:
I think there is some duplication here and the same snippet in `issueClaim`. Perhaps it makes sense to extract into a private method?